generator/generator.go

```diff
@@ -295,6 +296,10 @@ func (g *Generator) parseEnum(ts *ast.TypeSpec) (*Enum, error) {
 		enum.Prefix = g.Prefix + enum.Prefix
 	}
 
+	if g.BitField && enum.Type == "string" {
+		return nil, errors.New("bitfield enum only allows none string types")
```

Comment:
[nit] This error string needs to be worded differently.
```suggestion
		return nil, errors.New("bitfield option is not allowed on string types")
```